<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1333</width>
    <height>797</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Direct propagation</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="computeBButton">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>430</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Compute B field</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>470</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>861</width>
      <height>761</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="plotLayout"/>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>80</y>
      <width>381</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Flake</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QPushButton" name="loadShapeButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Shape</string>
     </property>
    </widget>
    <widget class="QPushButton" name="createShapeButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Create Shape</string>
     </property>
    </widget>
    <widget class="QLabel" name="magPhiLabel">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Phi (째)</string>
     </property>
    </widget>
    <widget class="QComboBox" name="magOriCombo">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>86</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="magThetaLine">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>140</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>90</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="magPhiLine">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>140</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="magThetaLabel">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>120</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Theta (째)</string>
     </property>
    </widget>
    <widget class="QLabel" name="magOriLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Orientation</string>
     </property>
    </widget>
    <widget class="QLabel" name="MsLabel">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>120</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Ms (muB/nm2)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="MsLine">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>140</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>24</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="includeFlakeCheck">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>40</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Include</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Magnetization</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>300</y>
      <width>341</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Dimensions</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLineEdit" name="gaussianSmoothLine">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>70</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="zLabel">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>z (nm)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="zLine">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>50.0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lxLine">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>15.0</string>
     </property>
    </widget>
    <widget class="QLabel" name="lyLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>ly (um)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lyLine">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>15.0</string>
     </property>
    </widget>
    <widget class="QLabel" name="lxLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>lx (um)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Gaussian
smooth (nm)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="selectFlakeCombo">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>40</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>500</y>
      <width>351</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Projection</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLineEdit" name="projPhiLine">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="projThetaLine">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>90</string>
     </property>
    </widget>
    <widget class="QLabel" name="projPhiLabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Phi (째)</string>
     </property>
    </widget>
    <widget class="QLabel" name="projThetaLabel">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Theta (째)</string>
     </property>
    </widget>
    <widget class="QLabel" name="projOriLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Orientation</string>
     </property>
    </widget>
    <widget class="QComboBox" name="projOriCombo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>86</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="BmaxLine">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>130</y>
       <width>81</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="BminLine">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>130</y>
       <width>81</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>B max</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>B min</string>
     </property>
    </widget>
    <widget class="QPushButton" name="centerZeroFieldButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>170</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Center 0B</string>
     </property>
    </widget>
    <widget class="QPushButton" name="autoscaleButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>170</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Autoscale</string>
     </property>
    </widget>
    <widget class="QPushButton" name="projectButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Plot projection</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveProjButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>210</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Save projection</string>
     </property>
    </widget>
    <widget class="QPushButton" name="reverseProjButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>60</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Reverse</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="addFlakeButton">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>40</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add flake</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1333</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
